src/chat/model_selector.rs

```diff
@@ -449,6 +466,17 @@ fn options_to_display(store: &Store) -> bool {
 }
 
 fn no_active_model(store: &Store) -> bool {
-    store.get_loaded_downloaded_file().is_none()
-        && store.chats.get_currently_loading_model().is_none()
+    store.get_loaded_downloaded_file().is_none() && store.get_loading_file().is_none()
+}
+
+fn rgb_color(r: u8, g: u8, b: u8) -> Vec4 {
```

Comment:
why is this another function? its only used on `hex_rgb_color`, why not just doing:
```rust
fn hex_rgb_color(hex: u32) -> Vec4 {
    vec4(
        ((hex >> 16) & 0xFF) as f32 / 255.0, // r
        ((hex >> 8) & 0xFF) as f32 / 255.0,  // g
        (hex & 0xFF) as f32 / 255.0,         // b
        1.0,                                 // a
    )
}
```

Reply:
also, i would move to utils or somthing like that